.gitignore

```diff
@@ -1 +1,2 @@
 /target
+/cargo-temp-bindings/cargo.lock
```

Comment:
You can ignore all cargo lock files since it duplicates `toml` file content in some part. No need to keep them since they're autogenerated.